src/test_l1_dsVideoDevice.c

```diff
@@ -1244,30 +1243,27 @@ void test_l1_dsVideoDevice_positive_dsGetFRFMode(void)
     int result;
     int index = 0;
     intptr_t handle = -1;
-    int fetchedFRFMode1;
-    int fetchedFRFMode2;
+    int fetchedFRFMode;
 
     // Step 01: Initialize video devices
     result = dsVideoDeviceInit();
     UT_ASSERT_EQUAL(result, dsERR_NONE);
 
     // Step 02: Obtain video device handle
-    result = dsGetVideoDevice(index, &handle);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
-    UT_ASSERT_TRUE(handle >= 0);
-
-    // Step 03: Get the FRF mode using the obtained handle
-    result = dsGetFRFMode(handle, &fetchedFRFMode1);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    for(int i = 0; i < gDSvideoDevice_NumVideoDevices; i++){
+        result = dsGetVideoDevice(index, &handle);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
+        UT_ASSERT_TRUE(handle >= 0);
 
-    // Step 04: Get the FRF mode using the obtained handle again
-    result = dsGetFRFMode(handle, &fetchedFRFMode2);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+        // Step 03: Get the FRF mode using the obtained handle
+        result = dsGetFRFMode(handle, &fetchedFRFMode);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
 
-    // Step 05: Compare the returned values
-    UT_ASSERT_EQUAL(fetchedFRFMode1, fetchedFRFMode2);
+        // Step 04: Compare the FRF mode with the value from the profile
+        UT_ASSERT_EQUAL(fetchedFRFMode, gDSVideoDeviceConfiguration[i].DefaultDFC);
+    }
 
-    // Step 06: De-initialize the video devices
+    // Step 05: De-initialize the video devices
     result = dsVideoDeviceTerm();
     UT_ASSERT_EQUAL(result, dsERR_NONE);
 
```

Comment:
in dsForceDisableHDRSupport test case the expected values `dsERR_OPERATION_NOT_SUPPORTED` as mention in header please update the same 
please check other API's also for the same 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsVideoDevice.h#L283

same with dsSetFRFMode , dsGetFRFMode & dsGetCurrentDisplayframerate() also check 
source expecting  `dsERR_OPERATION_NOT_SUPPORTED` 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsVideoDevice.h#L306

Reply:
done